lib/ansible/utils/module_docs_fragments/netapp.py

```diff
@@ -101,17 +111,22 @@ class ModuleDocFragment(object):
   username:
       required: true
       description:
-      - Please ensure that the user has the adequate permissions. For more information, please read the official documentation U(https://goo.gl/ddJa4Q).
+      - Please ensure that the user has the adequate permissions. For more information, please read the official documentation
+        U(https://mysupport.netapp.com/documentation/docweb/index.html?productID=62636&language=en-US).
+      aliases: ['user']
   password:
       required: true
       description:
       - Password for the specified user.
+      aliases: ['pass']
 
 requirements:
-  - solidfire-sdk-python (1.1.0.92)
+  - The modules were developed with SolidFire 10.1
+  - Ansible 2.7
```

Comment:
This line can be removed as the module ships with Ansible and our documentation is versioned